src/styles/settings/README.md

```diff
@@ -0,0 +1,67 @@
+## Spacings
+Paddings and margins are all in multiples of 5px from 5px to 50px for standard usage.
+
+## Responsive breakpoints
+| name | size |
+|:--|:--|
+| xs | < 576px |
+| sm | >= 576px |
+| md | >= 768px |
+| lg | >= 992px |
+| xl | >= 1200px |
+
+Code should be written for `xs` first, then modified using the `sm`, `md`, `lg` and `xl` breakpoints where appropriate.
+
+## Fonts
+### Font Family
+All non-monospace text uses Rubik, which can be found on Google fonts here: https://fonts.google.com/specimen/Rubik
+
+Monospace text uses Space Mono, available on Google Fonts here: https://fonts.google.com/specimen/Space+Mono
+
+### Font Sizes
+Font sizes differ between breakpoints. At md, lg and xl, the larger value will be used, and at xs and sm the smaller value will.
+
+| type | size | weight |
+|:--|:--|:--|
+| body | 1.8/1.6rem | 300 |
+| h1 | 3.6/2.4rem | 700 |
+| h2 | 2.8/2rem | 500 |
+| h3 | 2.2/2rem | 500 |
+| h4 | 1.8/1.6rem | 500 |
+
+## Colors
+
+These are the standard colours used for our branding, however the list is not exhaustive, and now colours may be introduced where appropriate.
```

Comment:
Typo - `new` not `now`

Reply:
Just can't keep away? 😛 Thanks! Fixed that now.